### ISSUE TITLE
How do you set a custom regex for private messages?

### ISSUE THREAD

**Issue by Sir-Will** [killjoy1221/TabbyChat-2#92]
I can't find where I can set a custom a custom regex for from and to private messages in the 1.12 tabbychat version.
In previous versions you could configure that in the tabbychat options.

**Comment by killjoy1221** [comment]
Private messages are now handled by a custom filter. Just specify that the channel starts with `@`. You can use regex groups to extract the recipient. It's a comma separated list, and unmatched groups will be skipped. e.g. `@$1, @$2`

**Comment by Sir-Will** [comment]
Like this?
![](https://i.imgur.com/nV6ezys.jpg)

Also what do I select as Message Pattern if I use custom regex?

**Comment by killjoy1221** [comment]
Yes, that will work. As for the message pattern, that doesn't matter. It
just acts as an additional filter. If it doesn't match, nothing will
happen. If you want to disable it anyway, just uncheck the box for it in
the server settings.

**Comment by Sir-Will** [comment]
Hm, doesn't work for me.
